app/controllers/concerns/idv/unavailable_concern.rb

```diff
@@ -0,0 +1,26 @@
+module Idv
+  class UnavailableError < StandardError; end
+
+  module UnavailableConcern
+    extend ActiveSupport::Concern
+
+    included do
+      rescue_from Idv::UnavailableError, with: :render_idv_unavailable
```

Comment:
This is a little different from how we handle vendor outage errors currently. I'd like to 503 rather than redirect to an error message so the user could refresh the page to try again.